route/auth-router.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Router = require('express').Router;
+const debug = require('debug')('ramble:auth-router');
+const jsonParser = require('body-parser').json();
+const parseBasicAuth = require('../lib/parse-basic-auth');
+const parseBearerAuth = require('../lib/parse-bearer-auth');
+
+const authController = require('../controller/auth-controller');
+
+const authRouter = module.exports = new Router();
+debug('authRouter');
+authRouter.post('/signup', jsonParser, function(req, res, next){
+  authController.signup(req.body)
+  .then(token => res.send(token))
+  .catch(next);
+});
+
+authRouter.get('/signin', parseBasicAuth, function(req, res, next){
+  authController.signin(req.auth)
+  .then(token => res.send(token))
+  .catch(next);
+});
+
+authRouter.put('/update/password', parseBearerAuth, jsonParser, function(req, res, next){
```

Comment:
this route should probbbly me named /user/password and be a put request

the fact that its a put route imples that its an update